laravel/database/factories/ScheduleFactory.php

```diff
@@ -1,20 +1,55 @@
 <?php
 
-use Carbon\Carbon;
+use App\Models\Department;
 use App\Models\Schedule;
+use App\Models\Shift;
+use Carbon\Carbon;
+use Faker\Generator as Faker;
 
-$factory->define(Schedule::class, function (Faker\Generator $faker, $data)
+$factory->define(Schedule::class, function (Faker $faker, array $data)
 {
-    $dates = json_decode($data['dates'], true);
+    if(env('APP_DEBUG') && !isset($data['department_id']))
+    {
+        Log::warning("Using Factory[Schedule] without setting department_id");
+    }
+
+    if(env('APP_DEBUG') && !isset($data['shift_id']))
+    {
+        Log::warning("Using Factory[Schedule] without setting shift_id");
+    }
+
+    $duration_min = 2; //hours
+    $duration_max = 8; //hours
+
+    $volunteer_min = 1;
+    $volunteer_max = 3;
+
+    $start_datetime = Carbon::tomorrow();
+    $end_datetime = $start_datetime->copy()->addWeeks($faker->randomDigitNotNull);
```

Comment:
Hmm, I wonder what will happen if a schedule is randomly created with an end date beyond the end date of the event it belongs to. :thinking: